hoomd/md/wall.py

```diff
@@ -479,7 +479,7 @@ def __repr__(self):
 #           *** Potentials ***
 
 
-class wallpotential(external._external_force):
+class wallpotential(external.field.Field):
```

Comment:
I think wall will inherit from a different class then `Field`, like an `External` base class in `hoomd/md/external/external.py`. Since this is not ported it is fine for now.